# HG changeset patch
# User Guillaume Flandin <guillaume.offline@gmail.com>
# Date 1543600963 -3600
#      Fri Nov 30 19:02:43 2018 +0100
# Node ID c27ecebf5b9b47d2dac311e07d628109aa1382eb
# Parent  46757ec0bac23db1018752350c8571f6ce61c031
struct2hdl.m: Add "uitable" to list of supported ui objects (patch #9084).

diff --git a/scripts/plot/util/struct2hdl.m b/scripts/plot/util/struct2hdl.m
--- a/scripts/plot/util/struct2hdl.m
+++ b/scripts/plot/util/struct2hdl.m
@@ -162,17 +162,17 @@ function [h, pout] = struct2hdl (s, p=[]
   elseif (strcmp (s.type, "image"))
     h = createimage (s, par);
   elseif (strcmp (s.type, "surface"))
     h = createsurface (s, par);
   elseif (strcmp (s.type, "hggroup"))
     [h, s, p] = createhg (s, p, par, hilev);
   elseif (any (strcmp (s.type, {"uimenu", "uicontextmenu",...
                                 "uicontrol", "uipanel", "uibuttongroup",...
-                                "uitoolbar", "uipushtool"})))
+                                "uitoolbar", "uipushtool", "uitable"})))
     if (isfield (s.properties, "extent"))
       s.properties = rmfield (s.properties, "extent");
     endif
     [h, s] = createui (s, par);
   else
     error ("struct2hdl: %s objects are not implemented yet", s.type);
   endif
 
